wheels/cloudauthz/meta.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+type: wheel
+name: cloudauthz
+purepy: true
```

Comment:
This is no longer needed unless Starforge fails to autodetect the wheel type.

Reply:
Cool